src/main/java/io/resourcepool/ssdp/client/SsdpParams.java

```diff
@@ -11,7 +11,12 @@
  */
 public class SsdpParams {
   private static InetAddress ssdpMulticastIpv4Address;
-  private static final int SSDP_MULTICAST_PORT = 1900;
+  private static   int SSDP_MULTICAST_PORT = 1900;
+  /**
+   * separate the message sending port and receiving port
+   * If another application on the same server also listens to udp port 1900, and the binding source port is 1900, the program may not receive a response
+   */
+  private static  int BIND_LOCAL_PORT=1900;
```

Comment:
Careful to your java coding conventions.
"=" needs spaces around, and non-final static attributes are camelCase.
To double-check those, run mvn clean install, there is a checkstyle plugin which will enforce you to respect basic java coding conventions :)